pkg/apiserver/manager/hwameistor/localstoragepool_controller.go

```diff
@@ -93,6 +94,9 @@ func (lspController *LocalStoragePoolController) makeStoragePoolNodesCollectionM
 		log.WithError(err).Error("Failed to list LocalStorageNodes")
 		return nil, err
 	}
+	sort.Slice(lsnList.Items, func(i, j int) bool {
+		return lsnList.Items[i].CreationTimestamp.String() > lsnList.Items[j].CreationTimestamp.String()
```

Comment:
we may use `return lsnList.Items[i].CreationTimestamp.After(lsnList.Items[j].CreationTimestamp.Time)` will be betther?

Reply:
thanks，i will update it ASAP